modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/toast/commands/OpenToast.es.js

```diff
@@ -25,25 +25,36 @@ const DEFAULT_RENDER_DATA = {
 
 const TOAST_AUTO_CLOSE_INTERVAL = 5000;
 
-const getDefaultAlertContainer = () => {
-	let container = document.getElementById(DEFAULT_ALERT_CONTAINER_ID);
+/**
+ * Function used to obtain the root element for rendering the Toast React component.
+ *
+ * When providing a container or a containerId, an element will be added as a first
+ * child of the element for preventing React replace the container/containerId DOM
+ * when rendering. Otherwise, the element will be appended to the body.
+ *
+ * When a container and containerId were provided, the container have precedence.
+ *
+ * @param {HTMLElement} container A container to be used to the Alert being positioned relatively.
+ * @param {string} containerId A containerId of the element to be opened relatively.
+ * @returns {HTMLElement} An element used to
```

Comment:
Rather than doing yet another round trip on this one, I'll push a commit fixing the grammar errors on top and then do a manual forward of this to get it out the way.

Reply:
Ok, I take that back. I'm not going to do a manual forward. The code is buggy. I'm going to fix it.